app/models/assignee.rb

```diff
@@ -0,0 +1,67 @@
+class Assignee
+  include ActiveModel::Model
+
+  attr_accessor :name, :lastname, :phone, :email, :profile_id, :access_code, :profile
+
+  # Method used to setup new Assignee with a user > profile/access_code
+  def self.setup(assignee_params, user)
+    new(assignee_params).prepare(user)
+  end
+
+  def is_present?
+    has_profile? || has_access_code?
+  end
+
+  def p_name
+    %(#{name} #{lastname})
+  end
+
+  def p_email
+    %("#{name} #{lastname}" <#{email}>)
+  end
+
+  def clear_attributes
+    self.name, self.lastname, self.phone = nil, nil, nil
+    self.email, self.profile_id, self.access_code = nil, nil, nil
+    attributes
+  end
+
+  def attributes
+    {
+        name: name, lastname: lastname, email: email,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.